src/themes/website/styles/widgets/widgets.scss

```diff
@@ -18,3 +18,29 @@
 @import "fui/operation-details.scss";
 @import "fui/apis-products-lists.scss";
 @import "fui/reports.scss";
+
+/**
+ * If you need to make sure that custom defined styles do not interfere with widgets,
+ * this is a good place to do it. Make sure to add !important to insure overriding.
+ */
+[data-bind="widget: widget"] {
+  .button {
+    margin: 10px 20px 10px 0 !important;
+    padding: 9px 50px !important;
+  }
+
+  .form-group {
+    margin: 5px 10px 20px 0 !important;
+    padding: 0 !important;
+
+    label {
+      margin: 0 0 0.5em !important;
+      padding: 0 !important;
+    }
+
+    .form-control {
+      margin: 0 !important;
+      padding: 7px 10px 5px !important;
+    }
+  }
+}
```

Comment:
Please, move all this to another file. Let's keep this for just imports.

Reply:
Why? This seems as an appropriate file to import all specific widget styles as well as defining global widget styles as well.

Reply:
This file is not redesign specific. Please, create a new one and place it there.
Also, why .fui-FluentProvider is specified 3 times? Can we specify it once as old widgets don't have this class at all.

Reply:
> This file is not redesign specific. Please, create a new one and place it there.

Where do we put `redesign specific` files?

> Also, why .fui-FluentProvider is specified 3 times?

Read a comment I written to it

Reply:
> Where do we put `redesign specific` files?
There is a folder named _fui_ right next to this file.

> Read a comment I written to it
Ok. My  bad. Let's leave like this.